Makefile

```diff
@@ -69,3 +69,9 @@ c: console
 # for running tests without entering container
 test:
 	docker exec -it testifi_app bash -c 'rspec -fd'
+
+
+# attach to Docker process running Rails so we can use byebug.
+# ctrl-P, ctrl-Q to detach.
```

Comment:
Doesn't `attach -it` solve this problem? I remember dealing with this a while ago.